VTEX - Marketplace Protocol - External Marketplace Mapper.json

```diff
@@ -264,32 +244,71 @@
                     }
                 ],
                 "requestBody": {
-                    "description": "",
                     "content": {
                         "application/json": {
                             "schema": {
-                                "$ref": "#/components/schemas/VtexMapperRegistration"
+                                "required": [
+                                    "displayName",
+                                    "categoryTreeEndPoint",
+                                    "mappingEndPoint",
+                                    "properties"
+                                ],
+                                "type": "object",
+                                "properties": {
+                                    "displayName": {
+                                        "type": "string",
+                                        "description": "Marketplace Name, that will be displayed in VTEX Mapper.",
+                                        "example": "Marketplace A"
+                                    },
+                                    "categoryTreeEndPoint": {
+                                        "type": "string",
+                                        "description": "Endpoint that returns categories and attributes according to VTEX  Mapper specifications.",
+                                        "example": "http://api.vtexinternal.com.br/api/{{marketplaceName}}/mapper/categories"
+                                    },
+                                    "mappingEndPoint": {
+                                        "type": "string",
+                                        "description": "Secure endpoint that will receive the category mapping sent by VTEX Mapper.",
+                                        "example": "http://api.vtexinternal.com.br/api/{{marketplaceName}}/mapper/mapping"
+                                    },
+                                    "properties": {
+                                        "type": "object",
+                                        "required": [
+                                            "allowsRemap"
+                                        ],
+                                        "description": "Refers to the `allowsRemap` property.",
+                                        "properties": {
+                                            "allowsRemap": {
+                                                "type": "boolean",
+                                                "description": "Allows rempaping categories in case the marketplace or seller tree is altered. If marked as `true`, all trees will be remapped.",
```

Comment:
```suggestion
                                                "description": "Allows remaping categories in case the marketplace or seller tree is altered. If marked as `true`, all trees will be remapped.",
```